README.md

```diff
@@ -1,17 +1,29 @@
-[![Build Status](https://travis-ci.org/cta-observatory/ctapipe_io_lst.svg?branch=master)](https://travis-ci.org/cta-observatory/ctapipe_io_lst)
+# ctapipe_io_lst [![Build Status](https://travis-ci.org/cta-observatory/ctapipe_io_lst.svg?branch=master)](https://travis-ci.org/cta-observatory/ctapipe_io_lst)
+
 
 EventSource Plugin for ctapipe, able to read LST zfits files
+and calibration them to R1 as needed for ctapipe tools.
 
 
-Install:
+Create a new environment:
 ```
 conda env create -n lstenv -f environment.yml
 conda activate lstenv
 pip install -e .
 ```
 
-Or in your favorite conda environment:
+Or just install in an existing environment:
 ```
-conda env update -f environment.yml
 pip install -e .
 ```
+
+
+## Test Data
+
+To run the tests, a set of non-public files is needed.
+If you are a member of LST, ask one of the project maintainers for the credentials
```

Comment:
This is a pity. I am not a member of LST .. so I understand from this point on I cannot run the test anymore.

Reply:
Not your fault ... also not an issue I have with this PR ... just a statement.

Reply:
A .. just a thought .. I guess it is possible to run part of the tests of ctapipe, even when one is not a member of LST right? Would it be possible to ... omit tests when the LST test files are not present, so that people who are not in LST can at least check that they did not mess up LSt unrelated stuff?

Reply:
Since the point of this package is testing a source for LST data, nearly all tests require the lst data...